src/CMakeLists.txt

```diff
@@ -99,122 +87,25 @@ endif()
-# ~~~
-# add_run_and_debug_targets(exampleApp)
-# ~~~
-#
-# short convenience target from cmake/cleanCpppExtensions.cpp
-# ~~~
-# add_custom_target(run DEPENDS run_exampleApp)
-# ~~~
-#
-# Manually specified convenience target for running the binary, we use macro
-# ~~~
-# add_custom_target(run_exampleApp
-#     WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/bin
-#     USES_TERMINAL
-#     COMMAND ./exampleApp
-#     DEPENDS exampleApp)
-# ~~~
-# And this would add -DBAR_ENABLE_FEATURE or equivalent to build command
-# ~~~
-# target_compile_definitions(exampleApp PRIVATE BAR_ENABLE_FEATURE)
-# ~~~
-#
-# Add compile time definition based on option option(OPTION_NAME "description for end user" OFF)
-# ~~~
-# if(OPTION_NAME)
-#     # you can do anything here, eg. setup new optional executable
-#     target_compile_definitions(exampleApp PRIVATE DEFINE_NAME)
-# endif()
-# ~~~
-#
-# Add building new shared library
-# ~~~
-# add_library(exampleLibrary SHARED ${LIBRARY_SOURCES})
-# target_link_libraries (exampleLibrary ${CMAKE_THREAD_LIBS_INIT})
-# set_property(TARGET exampleLibrary PROPERTY CXX_STANDARD 14) # we want C++14
-# ~~~
-#
-# How to include and link OpenGL and GLUT libraries
-# ~~~
-# find_package(OpenGL REQUIRED)
-# find_package(GLUT REQUIRED)
-# target_include_directories(exampleApp SYSTEM ${OPENGL_INCLUDE_DIRS} ${GLUT_INCLUDE_DIRS} )
-# target_link_libraries(exampleApp ${OPENGL_LIBRARIES} ${GLUT_LIBRARY} )
-# ~~~
-#
-# How to use some other standard packages
-# ~~~
-# find_package(ZLIB REQUIRED)
-# target_link_libraries (exampleApp PRIVATE ZLIB::ZLIB)
-#
-# find_package(OpenSSL REQUIRED)
-# target_link_libraries (exampleApp PRIVATE OpenSSL::SSL OpenSSL::Crypto)
-#
-# find_package(CURL REQUIRED)
-# target_link_libraries (exampleApp PRIVATE CURL::libcurl)
-# ~~~
-
-# ------------------------------------------------------------------------------
-# Unit tests
-
-# add_subdirectory(unittest)
-
 # -------------------------------------------------------------------------------
 # Copy MINGW needed libraries for building on windows
 
 if(MINGW)
     message(STATUS "MinGW detected")
     get_filename_component(GCC_PATH ${CMAKE_C_COMPILER} PATH)
     if(${GCC_PATH} MATCHES "mingw64/bin")
-        set(libgcc "libgcc_s_seh-1.dll") # 64bit
+        set(LIBGCC "libgcc_s_seh-1.dll") # 64bit
         message(STATUS "  64bit dlls is building")
     else()
-        set(libgcc "libgcc_s_dw2-1.dll") # 32bit
+        set(LIBGCC "libgcc_s_dw2-1.dll") # 32bit
         message(STATUS "  32bit dlls is building")
     endif()
 
-    install(FILES ${GCC_PATH}/${libgcc} ${GCC_PATH}/libstdc++-6.dll DESTINATION ./bin/)
+    install(FILES ${GCC_PATH}/${LIBGCC} ${GCC_PATH}/libstdc++-6.dll DESTINATION ./bin/)
 endif(MINGW)
 
 # ------------------------------------------------------------------------------
 # Other MISC targets
 
 # Adds misc targets: format, cppcheck, tidy, see cmake/CleanCppExtensions.cmake
 set(FORMAT_STYLE_FILENAME ${RADIUM_SRC_DIR}/scripts/clang-format)
```

Comment:
This `FORMAT_STYLE_FILENAME` was only used on the deleted macro `add_misc_targets`. I think it should be removed also

Reply:
right I’ll clean the comments also